javascript/elements/updates_for_element.js

```diff
@@ -40,22 +40,33 @@ export default class UpdatesForElement extends SubscribingElement {
     }
   }
 
+  shouldUpdate (data) {
```

Comment:
lol - this function is so gnarly.

Reply:
Yeah. Actually now that you say it this could be more self-documenting, i.e. split into 3 getters...

Reply:
Your call. I was just impressed at how it is like a homage to Perl.

Reply:
updated